Tests/scripts/pkg_dev_test_tasks.py

```diff
@@ -312,7 +312,10 @@ def main():
                 docker_image_created = docker_image_create(docker, requirements)
                 docker_run(project_dir, docker_image_created, args.no_test, args.no_pylint, args.keep_container, args.root)
         except subprocess.CalledProcessError as ex:
-            sys.stderr.write("[FAILED {}] Error: {}.\nOutput:\n{}\n".format(project_dir, str(ex), ex.output))
+            sys.stderr.write("[FAILED {}] Error: {}\n".format(project_dir, str(ex)))
+            if not LOG_VERBOSE:
```

Comment:
but this way if you face an exception you will not display its full contents

Reply:
It doesn't display anyway unless you run with -v because the way subprocesses are run. This fix seems to be sufficient.